### DISCUSSION TITLE
Remove "assigned to" and "shared with users" in detailViewTitle

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#14685] [How to] [answered]
I need to hide "assigned to" and "shared with users" from the module header, but that should be for certain role(s)
Anyone managed it in some not invasive way?

![image](https://user-images.githubusercontent.com/84110093/118578982-5bae9b80-b7b7-11eb-92d7-21141aaf8c5a.png)

**Comment by vovpff** [comment, marked as answer]
You need to change layout template for this custom module.
